vsintegration/src/FSharp.Editor/CodeFix/AddOpenCodeFixProvider.fs

```diff
@@ -133,6 +133,7 @@ type internal FSharpAddOpenCodeFixProvider
             
         let quilifySymbolFixes =
             candidates
+            |> Seq.filter (fun (entity,_) -> not(entity.FullRelativeName.Contains("op_"))) // Don't include fully-qualified operator names. The resultant suggestion makes the code not compile.
```

Comment:
Modified.  I'm not aware of another way (although I imagine there's something in the language service which we could use).